src/app/qgsattributerelationedit.cpp

```diff
@@ -6,22 +6,23 @@ QgsAttributeRelationEdit::QgsAttributeRelationEdit( const QString &relationid, Q
   mRelationId( relationid )
 {
   setupUi( this );
+  coCardinality->setToolTip( QStringLiteral( "For many to many relation, the direct link has to be selected. In-between table will be hidden." ) );
```

Comment:
tr

Reply:
"For many to many (N:M) relations..." or "For a many to many (N:M) relation..."
"The in-between..."